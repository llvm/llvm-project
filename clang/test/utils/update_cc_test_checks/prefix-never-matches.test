# RUN: cp -f %S/Inputs/prefix-never-matches.cpp %t.cpp
# RUN: %update_cc_test_checks %t.cpp
# RUN: diff -u %t.cpp %S/Inputs/prefix-never-matches.cpp.expected
