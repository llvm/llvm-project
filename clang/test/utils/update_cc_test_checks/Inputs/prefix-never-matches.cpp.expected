// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py
// RUN: %clang_cc1 -triple=x86_64-unknown-linux-gnu -emit-llvm -O0 -o - %s | FileCheck %s -check-prefix=A
// RUN: %clang_cc1 -triple=x86_64-unknown-linux-gnu -emit-llvm -O3 -o - %s | FileCheck %s -check-prefix=A

int foo(int i ) {
    return 1;
}
//// NOTE: These prefixes are unused and the list is autogenerated. Do not add tests below this line:
// A: {{.*}}
